app/assets/javascripts/discourse/app/components/composer-fullscreen-prompt.js

```diff
@@ -0,0 +1,30 @@
+import GlimmerComponent from "@glimmer/component";
+import I18n from "I18n";
+import { next } from "@ember/runloop";
+
+export default class ComposerFullscreenPrompt extends GlimmerComponent {
+  constructor() {
+    super(...arguments);
+    this.#setupFullscreenPrompt();
+  }
+
+  #setupFullscreenPrompt() {
+    next(() => {
+      const promptElement = document.querySelector(
+        ".composer-fullscreen-prompt"
+      );
```

Comment:
Instead of having to query for the element, you can use the `{{did-insert}}` and `{{will-destroy}}` modifiers 

https://guides.emberjs.com/release/upgrading/current-edition/glimmer-components/#toc_dom-manipulation